metanorma/Dockerfile.in

```diff
@@ -17,6 +17,8 @@ RUN cd /setup && bundle install
 # export java executable path
 ENV JAVA_HOME /usr/lib/jvm/default-java
 ENV PATH $PATH:$JAVA_HOME/bin
+ENV RUBYOPT -rbundler/setup
+ENV BUNDLE_GEMFILE /setup/Gemfile
```

Comment:
Why do we need to set these ENV variables in the image?

Reply:
In case when we install gem from git, those env vars need to make gem visible for ruby